core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2063,6 +2063,7 @@ class SparkContext(config: SparkConf) extends Logging {
    * @param jobId the job ID to cancel
    * @throws InterruptedException if the cancel message cannot be sent
```

Comment:
This is interesting too. This does not throws an exception unlike https://github.com/apache/spark/pull/16013/files#r89664921